### ISSUE TITLE
Need to confirm phone number twice

### ISSUE THREAD

**Issue by ihaid** [near/near-wallet#1152]
**Problem**
when 2FA is enabled with a phone number and you add the same phone to the recovery methods you receive 2 SMS with codes, one confirming adding a phone number and the second one for 2FA. Same for email.

**Expected Behavior**
When you creating an account and starting with phone number as a recovery and then adding the 2FA with the same phone number you have to confirm only once, so it should work the same.

**Steps to reproduce**

1. Go to /create page
2. Enter the account id
3. Activate seed phrase recovery
4. Enable 2FA with phone\email
5. Enable phone\email recovery method
6. You will receive 2 SMS to confirm

**Comment by vgrichina** [comment]
@kcole16 seems like very low priority, as during normal account creation you first setup recovery and only then 2FA. As I understand users can still complete the task as well (just slower). 

@ihaid does this only happen with phone or e-mail as well?

**Comment by ihaid** [comment]
> @ihaid does this only happen with phone or e-mail as well?

@vgrichina email as well, if for 2FA you have a phone then it happens with email for recovery and vice versa.
And yes, the user still can complete the task just will be bothered by waiting for two letters/SMS in a row.

**Comment by vgrichina** [comment]
>  if for 2FA you have a phone then it happens with email for recovery and vice versa.

@ihaid  hmm, not sure I understand this correctly. If you use 2FA with a phone but setup e-mail recovery, then it is expected that you both need to verify code on phone (to confirm 2FA) and code from e-mail (to confirm e-mail).

**Comment by ihaid** [comment]
> @ihaid hmm, not sure I understand this correctly. If you use 2FA with a phone but setup e-mail recovery, then it is expected that you both need to verify code on phone (to confirm 2FA) and code from e-mail (to confirm e-mail).

Sorry, it was a typo. I edit the comment, now everything is correct.

**Comment by jimmy3dita** [comment]
@ihaid can you check if this issue was fixed?

**Comment by ihaid** [comment]
> @ihaid can you check if this issue was fixed?

Yes, still exists
